### DISCUSSION TITLE
`SSG` without @vueuse/head

### DISCUSSION THREAD

**Discussion by vedxp** [antfu/vite-ssg#321] [Q&A]
I want to integrate ViteSSG in my vue project, but I don't want to use `@vueuse/head`.

When building, it gives error because I didn't install `@vueuse/head` as I don't want to.
I've tried installing `@vueuse/head` and the build succeeds with no error then.

Looking for ways to exclude `@vueuse/head` :)

---
Error:
>  ![Screenshot_20221130-165056~2](https://user-images.githubusercontent.com/87353286/204786161-a04b3a74-2762-44d0-b16c-84c59715a3b3.png)

**Comment by Gresdy** [comment]
您好, 您的邮件永宏已经收到！

**Comment by zhishui06** [comment]
嗯，收到了，谢谢！
